Source/Meadow.Foundation.Peripherals/Sensors.Environmental.DFRobotGravityDOMeter/Samples/DFRobotGravityDOMeter_Sample/DFRobotGravityDOMeter_Sample.csproj

```diff
@@ -9,7 +9,26 @@
     <AssemblyName>App</AssemblyName>
   </PropertyGroup>
   <ItemGroup>
+<<<<<<< HEAD
```

Comment:
these changes should be removed - we keep the develop branch using local refrences and main uses nugets